src/components/Button/index.js

```diff
@@ -6,14 +6,16 @@
 // https://github.com/alphagov/govuk_elements/blob/master/packages/govuk-elements-sass/public/sass/elements/_buttons.scss
 
 import glamorous from "glamorous";
+import React from "react";
+
 import {
   BUTTON_COLOUR,
   BUTTON_COLOUR_DARKEN_15,
   WHITE,
   YELLOW
 } from "govuk-colours";
 
-const Button = glamorous.button(
+const GButton = glamorous.button(
```

Comment:
was this supposed to be renamed?

Reply:
ah looks like it, although the naming is confusing. Maybe `GlamorousButton`? 👨‍💻

Reply:
yeah, react-docgen doesn't pick up glamorous components so I'm having to wrap them with a JSX component. I renamed the inner components to start with a G for Glamorous

Reply:
okay might be good to document that either inline comments or in the readme

Reply:
it's in the PR description and I've created an issue here https://github.com/penx/govuk-react/issues/81

Reply:
I agree it's not great, I just didn't want to spend too much time on documentation at this point, will hopefully revisit soon, maybe there will be progress on glamorous + react-docgen compatibility in the meantime and we can just undo this.

Reply:
ok sounds good